ckan/lib/cli.py

```diff
@@ -556,9 +556,6 @@ def rebuild_fast(self):
             package_ids.append(row[0])
 
         def start(ids):
-            ## load actual enviroment for each subprocess, so each have thier own
-            ## sa session
-            self._load_config()
```

Comment:
Is this really needed? I'd expect no changes in the old CLI

Reply:
i guess i've removed this by mistake